### PR TITLE
[SEMI-MODULAR] Adds catnip & felinid race drink

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Ports catnip over, making us able to add Frisky Kitty, the felinid race drink
image

Adds some comments that I forgot before.
Also fixes a bug with catnip tea.

## Why It's Good For The Game

Porting content good, bugfixes good

## Changelog
:cl:
add: Catnip! Grow cat crack
add: Frisky Kitty, the felinid race drink
fix: Fixes bug with catnip tea not metabolising, making you force-nya for the rest of the shift
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
will i learn not to have 10 merges in my commits? Maybe one day

**Comment by RatFromTheJungle** [comment]
denid for no swag
image
(hair isn't even green!)

**Comment by tf-4** [comment]
> 
> 
> denid for no swag
> image
> (hair isn't even green!)

Denial denied. This is going through. My next PR will add green hair dye

File: code/modules/unit_tests/plantgrowth_tests.dm [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	states |= icon_states('icons/obj/hydroponics/growing_mushrooms.dmi')
 	states |= icon_states('icons/obj/hydroponics/growing_vegetables.dmi')
 	states |= icon_states('goon/icons/obj/hydroponics.dmi')
+	states |= icon_states('modular_skyrat/master_files/icons/obj/hydroponics/growing.dmi') // SKYRAT ADDITION
```
Comment (RatFromTheJungle) [inline code/modules/unit_tests/plantgrowth_tests.dm:12]: what is growing

File: code/modules/vending/megaseed.dm [hunk 2]
```diff
@@ -45,6 +45,7 @@
 					/obj/item/seeds/wheat = 3,
 					/obj/item/seeds/whitebeet = 3)
 	contraband = list(/obj/item/seeds/amanita = 2,
+					/obj/item/seeds/tea/catnip = 3, // SKYRAT ADDITION
```
Comment (RatFromTheJungle) [inline code/modules/vending/megaseed.dm:48]: plant

File: modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 3]
```diff
@@ -33,3 +33,17 @@
 	if(C.mob_biotypes & MOB_ROBOTIC && C.blood_volume < BLOOD_VOLUME_NORMAL)
 		C.blood_volume += 0.5
 	..()
+
+// Catnip
+/datum/reagent/pax/catnip
+	name = "Catnip"
+	taste_description = "grass"
```
Comment (RatFromTheJungle) [inline modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/other_reagents.dm:40]: ew

**Review by RatFromTheJungle — APPROVED** [review]

File: modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 3 above)
Comment (tf-4) [reply modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/other_reagents.dm:40]:
```suggestion
	taste_description = "nice grass"
```
is this better

File: code/modules/unit_tests/plantgrowth_tests.dm (hunk 1 above)
Comment (tf-4) [reply code/modules/unit_tests/plantgrowth_tests.dm:12]: plant

File: modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 3 above)
Comment (RatFromTheJungle) [reply modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/other_reagents.dm:40]: yes

**Review by Gandalf2k15 — APPROVED** [review]
All systems are go, Captain. Engage.

**Pull request merged**